src/Symfony/Component/DependencyInjection/ParameterBag/FrozenParameterBag.php

```diff
@@ -28,8 +28,10 @@ class FrozenParameterBag extends ParameterBag
      *
      * @param array $parameters An array of parameters
      */
-    public function __construct(array $parameters = [])
-    {
+    public function __construct(
+        array $parameters = [],
+        protected array $deprecatedParameters = [],
```

Comment:
private would be fine
let's remove the `@param` annotation while at it

Reply:
Done.